packages/client/components/ReflectionGroup/ReflectionGroupTitleEditor.tsx

```diff
@@ -128,9 +128,11 @@ const ReflectionGroupTitleEditor = (props: Props) => {
   const {id: reflectionGroupId, title} = reflectionGroup
   const dirtyRef = useRef(false)
   const initialTitleRef = useRef(title)
-  const isLoading = title === ''
+
+  const isLoading = title === '' && !dirtyRef.current
```

Comment:
In master, if we remove the title while editing, it immediately shows the loading state. 

This is because in `EndDraggingMutation`, we initially set the title to an empty string while waiting for the ChatGPT title. While the title is an empty string, we show the loading ellipsis. 

This makes sure that the loading ellipsis only shows if the title is empty before the user begins editing.